staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -46,7 +46,6 @@ func newDelayingQueue(clock clock.Clock, name string) DelayingInterface {
 	ret := &delayingType{
 		Interface:       NewNamed(name),
 		clock:           clock,
-		heartbeat:       clock.NewTicker(maxWait),
```

Comment:
restore the heartbeat code, it serves as a backstop to ensure the workqueue doesn't get stuck